carmin.yaml

```diff
@@ -157,4 +305,63 @@ definitions:
         format: int32
       message:
         type: string
-    
\ No newline at end of file
+  execution:
+    type: object
+    required:
+      - name
+      - pipelineIdentifer
+      - status
+    properties:
+      name:
+        type: string
+        description: execution name
+      pipelineIdentifer:
+        type: string
+        description: which pipeline start this execution
+      status:
+        type: string
+        "enum": ["initializing",
+          "ready",
+          "running",
+          "finished",
+          "initialization_failed",
+          "execution_failed",
+          "unknown",
+          "killed"]
+  newExecution:
+    type: object
+    required:
+      - name
+      - pipelineIdentifer
+      - timeout
+      - inputValues
+    properties:
+      name:
+        type: string
+        description: name to display to users
+      pipelineIdentifer:
+        type: string
+      timeout:
+        type: integer
+        description: timeout in seconds
+      inputValues:
+        type: array
+        items:
+          $ref: '#/definitions/keyValuePair'
```

Comment:
I added several web-services for the execution operations. At the beginning, I was wondering how to add these execution operations in the `put` (it is defined in the google doc CARMIN REST). However, I find that it is really difficult to define them together (e.g. timeout, play, kill)  with swagger. So I added several web-services in addition. Hope it is OK for you @glatard @sapetnioc 

```
/studies/{study}/executions/{execution}/play
  get
/studies/{study}/executions/{execution}/kill
  get
```

I add two more information considering the stdout and stderr.

```
/studies/{study}/executions/{execution}/stdout
  get
/studies/{study}/executions/{execution}/stderr
  get
```